script/generate_incremental_table

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# propagate and exit on errors
+set -o pipefail -e
+
+# set UTC timezone by default
+export TZ="${TZ:-UTC}"
+
+COMMAND="'$(dirname "$0")/entrypoint' query --destination_table=${DESTINATION_TABLE:-\$DESTINATION_TABLE}'$'"
+COMMAND+='${DATE//-/} --parameter=submission_date:DATE:$DATE "$@"'
```

Comment:
Should we include `-n0 -q --replace`? Should we include `--time_partitioning_field=submission_date` to ensure we get time partitioning if this is the first time the script is being run?

Or is it expected that the user specifies such options as part of their invocation?

Reply:
I wasn't intending to, but testing shows me that all of these flags take the latest value given when specified multiple times, and those all seem like reasonable defaults, so yeah, i'll add them.